octopoes/octopoes/api/router.py

```diff
@@ -524,7 +527,38 @@ def recalculate_bits(octopoes: OctopoesService = Depends(octopoes_service)) -> i
 
     octopoes.commit()
 
-    return inference_count
+    RECALCULATE_BITS_SESSIONS[session_id] = inference_count
+
+
+@router.post("/bits/recalculate", tags=["Bits"])
+async def recalculate_bits(octopoes: OctopoesService = Depends(octopoes_service)) -> Any:
+    session_id = uuid.uuid4()
+    RECALCULATE_BITS_SESSIONS[session_id] = -1
+
+    asyncio.create_task(recalculate_bits_launcher(octopoes, session_id))
+
+    return {"session_id": session_id}
+
+
+@router.post("/bits/recalculate/status", tags=["Bits"])
+async def recalculate_bits_status(request: Request) -> Any:
+    try:
+        data = await request.body()
+    except Exception as e:
+        raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR, detail="Error receiving objects") from e
+    if data:
+        session_id = uuid.UUID(data.decode())
+        if session_id in RECALCULATE_BITS_SESSIONS:
+            retval = RECALCULATE_BITS_SESSIONS[session_id]
+            if retval > 0:
+                RECALCULATE_BITS_SESSIONS.pop(session_id)
+                return {"status": retval}
+            else:
+                return {"status": "awaiting"}
+        else:
+            return {"status": "unavailable"}
+    else:
+        return {"sessions": list(RECALCULATE_BITS_SESSIONS.keys())}
```

Comment:
This makes this endpoint both, in "Django-rest-framework-terms", a list() and get() at the same time. I'd say we should separate this into two endpoints (if we need actually need a list) and add the session_id perhaps as a url path parameter even for the "get" version.